CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 1.0.0 2025-09-05
```

Comment:
Let's take this out -- this should be discussed in the form of an issue first.  Opened one here, but in the meantime let's rebase against main: https://github.com/PhilanthropyDataCommons/service/issues/1895